datanode/pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>com.odysseusinc.arachne</groupId>
             <artifactId>execution-engine-commons</artifactId>
-            <version>2.0-DEV-SNAPSHOT</version>
+            <version>2.1-DEV-SNAPSHOT</version>
```

Comment:
There should be an official 2.0.0 release of ARACHNE Commons first
I don't understand this change

Reply:
That's right, with a small correction that it should be release of `execution-engine-commons`, not of arachne commons.
This will be updated as soon as that release occurs.